components/Fish.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { StyleSheet, Text, View, Image, TouchableOpacity } from "react-native";
+
+class Fish extends Component {
+  constructor({ navigation, fish }) {
+    super({ navigation, fish });
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <TouchableOpacity onPress={() => this.props.navigation.navigate("EditFish")} >
+            <Image 
+                style={styles.fishImage}
+                source={{
+                    uri: 'https://images.unsplash.com/photo-1594088979895-6086bd8d1f83?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80',
```

Comment:
This is just a pre-set image that will be changed once we have the user's image information being saved and sent back in our fetchFish call